cmd/minishift/cmd/start.go

```diff
@@ -161,6 +161,17 @@ func runStart(cmd *cobra.Command, args []string) {
 	libMachineClient := libmachine.NewClient(state.InstanceDirs.Home, state.InstanceDirs.Certs)
 	defer libMachineClient.Close()
 
+	if viper.GetString(configCmd.VmDriver.Name) == genericDriver {
+		if viper.GetString(configCmd.RemoteIPAddress.Name) == "" ||
+			viper.GetString(configCmd.RemoteSSHUser.Name) == "" ||
+			viper.GetString(configCmd.SSHKeyToConnectRemote.Name) == "" {
+			msg := fmt.Sprintf("Generic driver require additional information about IP address of remote machine, ssh key location and ssh username.\n"+
+				"Please enable experimental features of Minisift and provide following flags to use generic driver:\n"+
```

Comment:
`Enable experimental features of Minisift and provide following flags to use generic driver`